coapcloudconf.raml

```diff
@@ -32,8 +32,7 @@ traits:
 
```

Comment:
Why is oic.if.baseline the Default Interface?  I think oic.if.rw should be listed first in the enum for "if".

Reply:
Its aligned to current implementation.